src/templates/React InstantSearch widget/package.json.hbs

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "{{ packageName }}",
+  "version": "1.0.0",
+  "description": "{{ description }}",
+  "keywords": [
+    "instantsearch-widget",
+    "instantsearch",
+    "react-instantsearch",
+    "widget",
+    "connector",
+    "algolia"
+  ],
+  "license": "MIT",
+  "main": "./dist/index.cjs.js",
+  "module": "./dist/index.es.js",
+  "exports": {
+    "import": "./dist/index.es.js",
+    "require": "./dist/index.cjs.js"
+  },
+  "jsdelivr": "./dist/index.umd.js",
+  "unpkg": "./dist/index.umd.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "start": "vite",
+    "prebuild": "rm -rf dist",
+    "build": "vite build && npm run build:types",
+    "build:types": "tsc -p tsconfig.declaration.json",
+    "lint": "eslint --ext .js,.ts,.tsx .",
+    "lint:fix": "npm run lint --fix",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
+    "test:watch": "npm test -- --watch",
+    "test:types": "tsc",
+    "release": "standard-version"
+  },
+  "dependencies": {
+    "algoliasearch": "^4.9.1",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-instantsearch-dom": "^6.11.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.14.3",
+    "@babel/preset-env": "7.14.2",
+    "@babel/preset-react": "7.13.13",
+    "@babel/preset-typescript": "7.13.0",
+    "@testing-library/react": "11.2.7",
+    "@types/jest": "26.0.23",
+    "@types/node": "15.3.0",
+    "@types/react": "17.0.0",
+    "@types/react-dom": "17.0.0",
+    "@typescript-eslint/eslint-plugin": "4.24.0",
+    "@typescript-eslint/parser": "4.24.0",
+    "@vitejs/plugin-react-refresh": "1.3.1",
+    "babel-eslint": "10.1.0",
+    "babel-jest": "26.6.3",
+    "eslint": "7.26.0",
+    "eslint-config-algolia": "18.0.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-eslint-comments": "3.2.0",
+    "eslint-plugin-import": "2.23.2",
+    "eslint-plugin-jest": "24.3.6",
+    "eslint-plugin-jsdoc": "34.8.1",
+    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-prettier": "3.4.0",
+    "eslint-plugin-react": "7.23.2",
+    "eslint-plugin-react-hooks": "4.2.0",
```

Comment:
in the end, eslint gave us a bunch of devDependencies :(

Reply:
I think it's reasonable, the alternative is if the algolia eslint config comes with all those dependencies, but you're never sure that the dependency tree will be properly flat

Reply:
yeah this way, users are able to modify these to fit their needs/tastes.